integration/test_resources/help/cloud-account_iac-generate_gcp

```diff
@@ -0,0 +1,55 @@
+Use this command to generate Terraform code for deploying Lacework into an GCP environment.
+
+By default, this command will function interactively, prompting for the required information to setup the new cloud account. In interactive mode, this command will:
+
+* Prompt for the required information to setup the integration
+* Generate new Terraform code using the inputs
+* Optionally, run the generated Terraform code:
+  * If Terraform is already installed, the version will be confirmed suitable for use
+	* If Terraform is not installed, or the version installed is not suitable, a new version will be installed into a temporary location
+	* Once Terraform is detected or installed, Terraform plan will be executed
+	* The command will prompt with the outcome of the plan and allow to view more details or continue with Terraform apply
+	* If confirmed, Terraform apply will be run, completing the setup of the cloud account
+
+This command can also be run in noninteractive mode. See help output for more details on supplying required values for generation.
```

Comment:
how about: See help output for more details on the parameter value required for Terraform code generation.

Reply:
@JoshuaKim2021
what do you think about changing `the parameter value required` in your suggestion to `the parameter value(s) required`

Reply:
oh right. I agree with that.